modcc/symdiff.cpp

```diff
@@ -539,20 +539,52 @@ class ConstantSimplifyVisitor: public Visitor {
         e->rhs()->accept(this);
         expression_ptr rhs = result();
 
+        double
+            lval = expr_value(lhs),
+            rval = expr_value(rhs);
+
+        double rint = NAN;
```

Comment:
I believe the constant `NAN` is a float - use `std::numeric_limits<double>::quiet_NaN()` instead? `NAN` is used also in line 303 in this file.

Reply:
`std::nan("")`

Reply:
I used whatever was there before me.

Reply:
Converted to `std::nan`